depthai_nodes/ml/messages/classification.py

```diff
@@ -91,3 +94,26 @@ def top_score(self) -> float:
         @rtype: float
         """
         return self._scores[0]
+
+    @property
+    def transformation(self) -> dai.ImgTransformation:
+        """Returns the Image Transformation object.
```

Comment:
As the transformation property and setter are the same for every message, we could also make a message parent class that would define them once instead of copy-pasting them to all messages.

Reply:
Yes I also thought about it. I asked depthai team and when this will get ported, all messages will inherit from `dai.Buffer` and will have similar structure as `dai.ImgDetections` currently has. Thats why I left it as is.